libsrc/target/tiki100/stdio/generic_console.asm

```diff
@@ -216,22 +217,21 @@ done:
 
 
 generic_console_scrollup:
+	push	af
 	push	bc
 	push	de
+	ld	a,-8
+	call	gr_vscroll
 	call	swapgfxbk
-	ld	hl,1024
-	ld	de,0
-	ld	bc, 32768 - 1024
-	ldir
-	ld	h,d
-	ld	l,e
-	inc	de
+	ld	hl,$7C00
```

Comment:
I think this address is wrong?

![image](https://user-images.githubusercontent.com/394721/84822856-52e89f00-b015-11ea-88d8-7a51dd13311d.png)